nilearn/image/image.py

```diff
@@ -863,6 +863,11 @@ def clean_img(imgs, sessions=None, detrend=True, standardize=True,
         If True, the non-finite values (NaNs and infs) found in the images
         will be replaced by zeros.
 
+    mask_img: Niimg-like object
```

Comment:
Can you write that it is optional ?

Niimg-like object, optional

Reply:
Done :+1: